test/k8sT/Services.go

```diff
@@ -1860,7 +2175,7 @@ var _ = Describe("K8sServicesTest", func() {
 
 						BeforeAll(func() {
 							DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
-								"hostFirewall": "true",
+								"global.hostFirewall": "true",
```

Comment:
Is this correct? Why are we adding `global` here?

Reply:
Yeah, this is something I missed when rebasing the PR. The global prefix should not be here, I will remove these.
The test passes because we have backward compatibility for these helm variables in our CI.